File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0103U12LayDrugTreeInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS0103U12LayDrugTreeInfo
	{
		private String _code;
		private String _codeName;
		private String _code1;
		private String _codeName1;

		public String Code
		{
			get { return this._code; }
			set { this._code = value; }
		}

		public String CodeName
		{
			get { return this._codeName; }
			set { this._codeName = value; }
		}

		public String Code1
		{
			get { return this._code1; }
			set { this._code1 = value; }
		}

		public String CodeName1
		{
			get { return this._codeName1; }
			set { this._codeName1 = value; }
		}

		public OCS0103U12LayDrugTreeInfo() { }

		public OCS0103U12LayDrugTreeInfo(String code, String codeName, String code1, String codeName1)
		{
			this._code = code;
			this._codeName = codeName;
			this._code1 = code1;
			this._codeName1 = codeName1;
		}

	}
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0103U12LayDrugTreeInfo.java
package nta.med.data.model.ihis.ocsa;

public class OCS0103U12LayDrugTreeInfo {
	private String code;
	private String codeName;
	private String code1;
	private String codeName1;

	public OCS0103U12LayDrugTreeInfo(String code, String codeName,
			String code1, String codeName1) {
		super();
		this.code = code;
		this.codeName = codeName;
		this.code1 = code1;
		this.codeName1 = codeName1;
	}

	public String getCode() {
		return code;
	}

	public void setCode(String code) {
		this.code = code;
	}

	public String getCodeName() {
		return codeName;
	}

	public void setCodeName(String codeName) {
		this.codeName = codeName;
	}

	public String getCode1() {
		return code1;
	}

	public void setCode1(String code1) {
		this.code1 = code1;
	}

	public String getCodeName1() {
		return codeName1;
	}

	public void setCodeName1(String codeName1) {
		this.codeName1 = codeName1;
	}

}
